modules/oxide/src/less/theme/content/diff/diff.less

```
@@ -17,41 +17,66 @@
   .tox-@{pluginName}__annotation--added,
```

Stylelint (17.6.0)

[error] Unknown word pluginName (CssSyntaxError)

(CssSyntaxError)